libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -730,6 +730,13 @@ AP_InertialSensor::detect_backends(void)
         _add_backend(AP_InertialSensor_Invensense::probe(*this, hal.spi->get_device(HAL_INS_MPU9250_NAME), ROTATION_ROLL_180));
         break;
 
+    case AP_BoardConfig::PX4_BOARD_AUAV21:
+        // AUAV2.1 uses ICM20608 on the ACCEL_MAG device and a MPU9250 on the old MPU6000 CS line
+        _fast_sampling_mask.set_default(3);
```

Comment:
Should the fast sampling be enabled in both? Won't the use of the compass of MPU9250 forbid the use of fast sampling?